oci/defaults.go

```diff
@@ -154,6 +154,10 @@ func DefaultLinuxSpec() specs.Spec {
 			{Type: "pid"},
 			{Type: "ipc"},
 		},
+		// Always use MS_PRIVATE for rootfs propagation to make sure that we
+		// don't leak any MS_UNBINDABLE mounts into the container's namespace
+		// while its resolving volume bindmounts.
+		RootfsPropagation: "private",
```

Comment:
I will have to look at the logic more closely. I am forgetting the details. So this will apply "rprivate" on "/" in container mount namespace? If yes, this will break "shared/slave" volume mounts? "-v /mnt/vol1:/foo:slave".

Reply:
Can we not have private on the graphdriver root as well as unbindable?